mesh/nurbs.hpp

```diff
@@ -932,6 +938,12 @@ class ParNURBSExtension : public NURBSExtension
        The @a parent can be either a local NURBSExtension or a global one. */
    ParNURBSExtension(NURBSExtension *parent,
                      const ParNURBSExtension *par_parent);
+
+   ParNURBSExtension(NURBSExtension *parent,
+                     Array<NURBSExtension *> VNURBSExt,
+                     const ParNURBSExtension *par_parent);
+
+   Array<int> GetPartitioning() {return partitioning;}
```

Comment:
This is never called, so it can be removed (unless there is a good reason to keep it).